src/tests/dune

```diff
@@ -25,6 +25,7 @@
     osemgrep_targeting
     osemgrep_language_server
   )
+ (modes native) ; bytecode here for debugging
```

Comment:
so why did you remove the bytecode from the modes?

Reply:
Forgot to remove the comment. Going to revisit this today in a separate PR (wanted to get this merged for andre)